src/components/forms/formComponents.js

```diff
@@ -45,11 +49,13 @@ const normalizeValue = (value) => {
 };
 
 const onFormInputChange = (config, value) => {
```

Comment:
I'm getting a TypeError on the data_submission_form page when I get to the Data Access Governance, Data Access Committee section and I choose a DAC from the dropdown and then use the "X" to cancel my selection. See screenshot of error and stack trace:
<img width="1114" alt="TypeError" src="https://user-images.githubusercontent.com/59845076/189449412-cc4a83f3-71a6-49cd-827a-19549592ab43.png">
<img width="715" alt="TypeError Stack Trace" src="https://user-images.githubusercontent.com/59845076/189449422-ce977642-6796-4796-a85b-96ffc57c3cc9.png">

Reply:
fixed!!